lms/envs/common.py

```diff
@@ -2677,3 +2677,6 @@
     'options': {},
 }
 PROCTORING_SETTINGS = {}
+
+# Grade book: max students per page
+MAX_STUDENTS_PER_PAGE_GRADE_BOOK = 25
```

Comment:
We should just make one UX decision here about how many students are on a page; this does not need to be configurable per instance.

Yes, making a constant in api.py would make sense. I would think that ccx/view.py would import the constant.